tests/phpunit/tests/test-switcher.php

```diff
@@ -186,4 +189,28 @@ function test_dropdown() {
 		$this->assertNotEmpty( $xpath->query( '//select/option[.="Français"]' )->length );
 		$this->assertNotEmpty( $xpath->query( '//script' )->length );
 	}
+
+	function test_with_hide_if_no_translation_option_in_admin_context() {
+		$en = $this->factory->post->create();
+		self::$model->post->set_language( $en, 'en' );
+
+		$fr = $this->factory->post->create();
+		self::$model->post->set_language( $fr, 'fr' );
+
+		self::$model->clean_languages_cache(); // FIXME for some reason, I need to clear the cache to get an exact count
+		$this->admin->links->curlang = self::$model->get_language( 'en' );
```

Comment:
Why do you need a `$curlang` property?

Reply:
We do have a `curlang` in the widget editor which is the default language, but this is not needed here.